custom/icds/tests/test_custom_rules.py

```diff
@@ -230,6 +233,34 @@ def assert_person_case_is_under_N_years_old(self, n_years, criteria_name):
             case = self._set_dob(case, '2018-02-22')
             self.assertFalse(rule.criteria_match(case, dob + relativedelta(days=90)))
 
+    def test_ccs_record_case_has_future_edd(self):
+        rule = _create_empty_rule(self.domain, case_type='ccs_record')
+        rule.add_criteria(CustomMatchDefinition,
+            name='ICDS_CCS_RECORD_CASE_HAS_FUTURE_EDD')
+
+        def check(case, edd, match):
+            case = self._set_case_props(case, {"edd": edd})
+            (self.assertTrue if match else self.assertFalse)(
+                rule.criteria_match(case, now),
+                "%s case with edd=%s should%s match" % (
+                    case.type, edd, "" if match else " not",
+                )
+            )
+
+        now = datetime(2018, 2, 22, 12, 0)
+        with _with_case(self.domain, 'ccs_record', datetime.utcnow()) as case:
```

Comment:
would be good to also just add a line here for `self.assertFalse(rule.criteria_match(case, now))` to test the scenario where edd is not yet set

Reply:
655ebddc6d862b4da996767d1e81350b6d4505d7